src/components/common/TmSessionWelcome.vue

```diff
@@ -12,6 +12,13 @@
 
       <div class="tm-session-main">
         <LiSession
+          v-if="!checkBrowser"
```

Comment:
@faboweb Just wondering if this approach is correct. I feel like it's kinda clunky but it gets the job done. 

I was thinking about modifying the `LiSession` component to achieve the same effect but it seemed like I could spend my time on other more pressing issues and leave this to be refactored at a later time.